src/main/scala/mesosphere/marathon/core/task/tracker/impl/TaskCreationHandlerAndUpdaterDelegate.scala

```diff
@@ -3,50 +3,54 @@ package mesosphere.marathon.core.task.tracker.impl
 import akka.actor.ActorRef
 import akka.util.Timeout
 import mesosphere.marathon.core.base.Clock
-import mesosphere.marathon.core.task.Task
+import mesosphere.marathon.core.task.TaskStateOp.ReservationTimeout
+import mesosphere.marathon.core.task.{ TaskStateChange, TaskStateOp, Task }
 import mesosphere.marathon.core.task.tracker.impl.TaskTrackerActor.ForwardTaskOp
-import mesosphere.marathon.core.task.tracker.{ TaskCreationHandler, TaskTrackerConfig, TaskUpdater }
-import mesosphere.marathon.state.PathId
-import org.apache.mesos.Protos.TaskStatus
+import mesosphere.marathon.core.task.tracker.{
+  TaskReservationTimeoutHandler,
+  TaskStateOpProcessor,
+  TaskCreationHandler,
+  TaskTrackerConfig
+}
 
 import scala.concurrent.Future
 import scala.concurrent.duration._
 import scala.util.control.NonFatal
 
 /**
-  * Implements the [[TaskCreationHandler]]/[[TaskUpdater]] traits by sending messages to the TaskTracker actor.
+  * Implements the [[TaskStateOpProcessor]] trait by sending messages to the TaskTracker actor.
```

Comment:
`TaskOpProcessor` and `TaskStateOpProcessor` are very confusingly named since they do very similar things.

Is there a cleanup/renaming issue that already tracks this?